app/ante/evm/09_gas_consume.go

```diff
@@ -241,3 +242,24 @@ func CheckBlockGasLimit(ctx sdk.Context, gasWanted uint64, minPriority int64) (s
 
 	return ctx, nil
 }
+
+// UpdateTxFee updates the cumulative transaction fee
+func UpdateTxFee(
+	cumulativeTxFee sdk.Coins,
+	msgFee *big.Int,
+	denom string,
+) sdk.Coins {
+	return cumulativeTxFee.Add(
+		sdk.Coin{
+			Denom:  denom,
+			Amount: sdkmath.NewIntFromBigInt(msgFee),
+		},
+	)
+}
+
+// UpdateGasLimit updates the cumulative gas limi
+func UpdateGasLimit(
+	txGasLimit, gasLimit uint64,
+) uint64 {
+	return txGasLimit + gasLimit
+}
```

Comment:
no need for this function, it uses more lines than just using `txGasLimit + gasLimit`